src/constants/constants.ts

```diff
@@ -64,25 +64,26 @@ const sampleBody = `
  `;
 
 export const newSection: Post = {
-  _id: 'mock_post_id',
+  id: '',
   title: '**replace with title',
   body: sampleBody,
   tags: Array<Tag>(),
+  doc_ref: '',
```

Comment:
I dont think we need a doc_ref on posts, I think its just an array of dof_refs on the guide